### PR TITLE
Hydro devel

### PR DESCRIPTION
**Pull request by sachinchitta**
This pull request adds debug flag to the launch file and changes some WARN statements to DEBUG.

### CODE REVIEW

File: nodes/objects_database_node_simple.cpp [hunk 1]
```diff
@@ -401,22 +398,13 @@ class ObjectsDatabaseNode
         }
         else
         {
-	  // AWFUL HACK
-	  std::vector<std::string> joint_names_l;
```
Comment (mateiciocarlie) [inline nodes/objects_database_node_simple.cpp:405]: Wait - why was this part removed?

File: nodes/objects_database_node_simple.cpp [hunk 2]
```diff
@@ -401,22 +398,13 @@ class ObjectsDatabaseNode
...
-	  joint_names_l.push_back(joint_names[3]);
-	  joint_names_l.push_back(joint_names[4]);
-	  joint_names_l.push_back(joint_names[5]);
-	  //	  for(std::size_t k =0; k < joint_names.size(); ++k)
-	  //	    ROS_INFO("Joint names: %d %s", k, joint_names[k].c_str());
-	  joint_names = joint_names_l;
           //unfortunately we have to hack this, as the grasp is really defined by a single
           //DOF, but the urdf for the PR2 gripper is not well set up to do that
-          if ( joint_names.size() != 4 || (*it)->final_grasp_posture_.get().joint_angles_.size() != 1)
+	  /*          if ( joint_names.size() != 4 || (*it)->final_grasp_posture_.get().joint_angles_.size() != 1)
```
Comment (mateiciocarlie) [inline nodes/objects_database_node_simple.cpp:403]: ... and this?

Comment (sachinchitta) [inline nodes/objects_database_node_simple.cpp:403]: This is no longer necessary because things are handled the right way downstream by the moveit gripper plugin for the PR2.
